app/models/concerns/iu_user_roles.rb

```diff
@@ -1,4 +1,4 @@
-module PulUserRoles
```

Comment:
We need to find a way to configure-in site-specific modules and separate them from common code.